src/DependencyInjection/Configuration.php

```diff
@@ -59,7 +59,7 @@ public function getConfigTreeBuilder()
                             ->addDefaultsIfNotSet()
                             ->children()
                                 ->booleanNode('enabled')
-                                    ->defaultTrue()
+                                    ->defaultFalse()
```

Comment:
As doctrine is still a hard dependency, we can leave this to true (or remove it as an hard dependency?)